pkg/build/builder/source.go

```diff
@@ -392,12 +393,18 @@ func extractSourceFromImage(ctx context.Context, dockerClient DockerClient, stor
 		}
 	}
 
+	defaultContainerConfig, err := cconfig.Default()
+	if err != nil {
+		return err
+	}
```

Comment:
Does this have a high risk of erroring out here? Would we be able to easily identify errors from here?

Reply:
After a quick spin through the code, it looks like we get enough detail in error results that we can usually pin down where an error occurred, if one does.
Missing `containers.conf` and `libpod.conf` files shouldn't create errors, though if we're unprivileged and we can't determine where the file should be, we get an error, but we're privileged here.
I think we're pretty well insulated from having new surprise errors if we pull this in.